### PR TITLE
External connector

### PR DESCRIPTION
**Pull request by grzegorzswistak**

### CODE REVIEW

**Comment by boly38** [comment]
For open public project you could enable [travis-ci for java project](https://blog.frankel.ch/travis-ci-tutorial-for-java-projects/) and append mvn clean install (and maybe test) stage

File: src/main/java/com/orange/lo/sdk/Main.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.orange.lo.sdk;
+
+import com.orange.lo.sdk.externalconnector.DataManagementExtConnector;
+import com.orange.lo.sdk.externalconnector.model.DataMessage;
+import com.orange.lo.sdk.externalconnector.model.Metadata;
+import com.orange.lo.sdk.externalconnector.model.NodeStatus;
+import com.orange.lo.sdk.externalconnector.model.Status;
+import com.orange.lo.sdk.externalconnector.model.Value;
+
+public class Main {
```
Comment (szlisiecki) [inline src/main/java/com/orange/lo/sdk/Main.java:10]: We don't need this class

**Review by szlisiecki — APPROVED** [review]

**Comment by grzegorzswistak** [comment]
> For open public project you could enable [travis-ci for java project](https://blog.frankel.ch/travis-ci-tutorial-for-java-projects/) and append mvn clean install (and maybe test) stage [by creating `.travis.yml` at root level]

But why Travis? We have two actions needed to build a project. One that uses similar commands to the ones you just listed, and the other one to automatically release the version using the maven release plugin. I'm not sure what more could Travis give us?

**Comment by boly38** [comment]
> > For open public project you could enable [travis-ci for java project](https://blog.frankel.ch/travis-ci-tutorial-for-java-projects/) and append mvn clean install (and maybe test) stage [by creating `.travis.yml` at root level]
> 
> But why Travis? We have two actions needed to build a project. One that uses similar commands to the ones you just listed, and the other one to automatically release the version using the maven release plugin. I'm not sure what more could Travis give us?

My point was only to have a pipeline on PR, but I didn't see that you already have one. Solved. Sorry for the noise :) 👍

**Comment by grzegorzswistak** [comment]
> > > For open public project you could enable [travis-ci for java project](https://blog.frankel.ch/travis-ci-tutorial-for-java-projects/) and append mvn clean install (and maybe test) stage [by creating `.travis.yml` at root level]
> > 
> > 
> > But why Travis? We have two actions needed to build a project. One that uses similar commands to the ones you just listed, and the other one to automatically release the version using the maven release plugin. I'm not sure what more could Travis give us?
> 
> My point was only to have a pipeline on PR, but I didn't see that you already have one. Solved. Sorry for the noise :) +1

No problem :) Your comments are appreciated :)

**Pull request merged**